tests/unit/orcid/test_orcid_converter.py

```diff
@@ -135,8 +135,8 @@ def test_successfull_conversion():
                                                                              'contributor-sequence': 'ADDITIONAL'},
```

Comment:
You are adding quite a bit of untested code in this PR, so once it is more stable would be good to add them.